app/javascript/components/organisms/StatusModalWrapper.js

```diff
@@ -10,27 +11,35 @@ import { TextAreaField, Label, FieldError } from 'utils/form/final_form/Field'
 import validate from 'utils/form/validations'
 import {
   statusRequiresComments,
+  LEASE_UP_STATUS_VALUES,
   LEASE_UP_SUBSTATUS_OPTIONS,
+  LEASE_UP_SUBSTATUS_VALUES,
   validateStatusForm
 } from 'utils/statusUtils'
 
 import FormModal from './FormModal'
 
 const StatusModalWrapper = ({
-  isOpen,
-  onClose,
-  onSubmit,
-  showAlert,
-  alertMsg,
-  onAlertCloseClick,
-  loading,
+  alertMsg = null,
+  isOpen = false,
+  loading = false,
+  onAlertCloseClick = () => {},
+  onClose = () => {},
+  onSubmit = () => {},
+  showAlert = false,
   status = null,
+  submitButton,
   subStatus = null,
-  header,
-  submitButton
+  numApplicationsToUpdate = 1,
+  title = null
 }) => (
   <FormModal
-    header={header}
+    title={title}
+    subtitle={
+      numApplicationsToUpdate > 1
```

Comment:
I'm not sure if this is the right logic because you can check only 1 checkbox. That's why I had the bulk update flag in there

Reply:
Yeah sorry I realized that a little too late. Okay so adding back the boolean flag should let us fix this and the other thing I mentioned above, where we don't want to deselect any of the checkboxes if it's _not_ a bulk update operation.

Reply:
I can add it.